langfuse/callback.py

```diff
@@ -524,7 +538,9 @@ def on_llm_end(
                 )
 
                 self.runs[run_id].state = self.runs[run_id].state.update(
-                    UpdateGeneration(completion=extracted_response, end_time=datetime.now(), usage=llm_usage)
+                    UpdateGeneration(
+                        completion=extracted_response, end_time=datetime.now(), usage=llm_usage, metadata=token_metadata
```

Comment:
Why do you want to put the token metadata on the metadata as well? We are already tracking that in `usage`.

Reply:
I thought of giving all the information in metadata. But will remove it because in `usage` it's included.

Reply:
Awesome